src/flex/Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN set -xe \
   && /contrast/service/x86_64/contrast-flex-service --comms-parent-dir "/contrast" --agents-parent-dir "/contrast" agents \
   && /contrast/service/x86_64/contrast-flex-service --comms-parent-dir "/contrast" --agents-parent-dir "/contrast" auto-attach set true
 
+# Adjust the absolute path provided in PHP agent ini files
+RUN find /usr/bin/contrast-flex-agent/flex-agents/php/*/*/contrast.ini -type f -exec sed -i 's|/usr/bin/contrast-flex-agent/|/contrast/|g' {} \;
```

Comment:
Do we need `set -xe \` in this command? The other `RUN`'s have it. Might be worth adding for consistency even if it's not a requirement.

Reply:
Should the find be `find /usr/bin/contrast-flex-agent/flex-agents/php/*/*/contrast_ini/contrast.ini` ? I think the `contrast_ini` file is missing.

Reply:
```
find /usr/bin/contrast-flex-agent/flex-agents/php/*/*/contrast_ini/contrast.ini -type f -exec sed -i 's|/usr/bin/contrast-flex-agent/|/contrast/|g' {} +
```
works locally.